Source/PropertyTools.Wpf/DataGrid/Definitions/PropertyDefinition.cs

```diff
@@ -299,7 +301,26 @@ public void UpdateDescriptor(Type itemType)
         /// </summary>
         protected void SetEnumItemsSource()
         {
-            this.ItemsSource = Enum.GetValues(this.PropertyType);
+            var values = new List<object>();
+            var enumType = this.PropertyType;
+            if (enumType != null)
+            {
+                var ult = Nullable.GetUnderlyingType(enumType);
+                if (ult != null)
+                {
+                    // TODO: this does not work properly, do we need to use a converter?

```

Comment:
I think using `GetUnderlyingType` is cleaner.
The problem is how to show the `null` value in the combo box. Try the example to see the issue. I think a converter or custom item template is needed!